### DISCUSSION TITLE
Searching in file sometimes switches to insert mode and enters text at cursor

### DISCUSSION THREAD

**Discussion by hubertlepicki** [LazyVim/LazyVim#3151] [Q&A] [answered]
There is some configuration or a plugin, which triggers the unexpected to me behavior that is causing me to make accidental edits in files I search for content in.

What I do is that I open a file, and then want to search for something inside, for example "asdfasdfasdf". I do it by hitting forward slash, followed by the text I want to search for, for example "/asdfasdfasdf". If I do that, *sometimes* this however switches to insert mode and start inserting the text I want to search at cursor.

This is not happening always, but frequent enough for me to get annoyed.

Anyone knows what can be causing this and how can I disable it and use the default behavior?

https://github.com/LazyVim/LazyVim/assets/33274/db9b27f7-ca3a-4263-8d33-19b4746620f7

**Comment by dpetka2001** [comment, marked as answer]
It's same as #2514.

**Comment by hubertlepicki** [comment]
@dpetka2001 thank you so much. I disabled flash.nvim by creating this file at `~/.config/nvim/lua/plugins/disabled.lua`:

```
return {
  { "folke/flash.nvim", enabled = false },
}
```

**Reply by folke** [reply]
fyi, I actually made an update to flash earlier today that disables flash during search by default.

**Comment by nipuna-perera** [comment]
This was tripping a lot of people. Thanks for disabling in search by
default. It's a welcome change.
